src/web/server/htmlTemplate.ts

```diff
@@ -68,6 +68,10 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // This tag enables pages to be featured in Google Discover as large previews
+    // See: https://developers.google.com/search/docs/advanced/mobile/google-discover?hl=en&visit_id=637424198370039526-3805703503&rd=1
+    const googleDiscoverMetaTag = '<meta name="robots" content="max-image-preview:large">';
```

Comment:
We could just inline the string rather than assigning it to a variable?

Reply:
No problem, I was just following the style there, but yeah it'll be fine as a string.